app/src/main/java/org/oppia/android/app/help/HelpActivity.kt

```diff
@@ -7,10 +7,14 @@ import org.oppia.android.R
 import org.oppia.android.app.activity.InjectableAppCompatActivity
 import org.oppia.android.app.drawer.NAVIGATION_PROFILE_ID_ARGUMENT_KEY
 import org.oppia.android.app.help.faq.FAQListActivity
+import org.oppia.android.app.help.thirdparty.ThirdPartyDependencyListActivity
 import javax.inject.Inject
 
 /** The help page activity for FAQs and feedback. */
-class HelpActivity : InjectableAppCompatActivity(), RouteToFAQListListener {
+class HelpActivity :
+  InjectableAppCompatActivity(),
+  RouteToFAQListListener,
+  RouteToThirdPartyDependencyListListener {
```

Comment:
Add one blank line after this line as the @Inject line is not clearly indicated as separated line because of indentation issue.

Reply:
Done